src/pages/Profile.js

```diff
@@ -74,7 +74,7 @@ const styles = {
     },
     linkContentTop: hasBottom => ({
       display: 'grid', rowGap: '0.6rem',
-      backgroundColor: colors.light(), padding: '1.2rem',
+      backgroundColor: colors.light(0.2), padding: '1.2rem',
```

Comment:
Can you provide before/after screenshots for Terra so I know I'm looking at the right thing?

Reply:
Sure. 

### Before
![image](https://user-images.githubusercontent.com/5438223/183165579-bb7f8992-64ec-46f4-bb7a-0d9c1b1762c5.png)


### After
![image](https://user-images.githubusercontent.com/5438223/183166850-0ddeeab8-2dd7-4ea9-ad7a-6235be4d17d4.png)